packages/adapter-netlify/index.js

```diff
@@ -55,8 +54,10 @@ export default function (options) {
 			utils.copy_client_files(publish);
 
 			utils.log.minor('Writing redirects...');
-			utils.copy('_redirects', `${publish}/_redirects`);
-			appendFileSync(`${publish}/_redirects`, '\n\n/* /.netlify/functions/render 200');
+
+			const redirectPath = join(publish, '_redirects');
+			utils.copy('_redirects', redirectPath);
+			appendFileSync(redirectPath, '\n\n/* /.netlify/functions/__render 200');
```

Comment:
does this need to be `functions-internal` instead of `functions`?

Reply:
No, the public path is still `functions`